src/help/zaphelp/contents/start/concepts/sessionManagement.html

```diff
@@ -26,6 +26,13 @@
 		the <a href="httpsessions.html">Http Sessions</a> Extension.
 	</p>
 
+	<h3>
+		<a name="hasm">Http Authentication Session Management</a>
```

Comment:
Tks! Seems mixed within the page, if you view full changes its all caps some places and not elsewhere :(